docker-images/10.5/server/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright (c) 2011-2019 Software AG, Darmstadt, Germany and/or Software AG USA Inc., Reston, VA, USA, and/or its subsidiaries and/or its affiliates and/or their licensors.
+# Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG.
+
+if [ "$ACCEPT_EULA" == "Y" ]; then
+    echo "You accepted license agreement"
+else
+    echo "You must accept license agreement with ACCEPT_EULA=Y environment variable. Exiting..."
+    exit 10
+fi
+
+sed -i -r 's/OFFHEAP_RESOURCE1_NAME/'$OFFHEAP_RESOURCE1_NAME'/; s/OFFHEAP_RESOURCE1_UNIT/'$OFFHEAP_RESOURCE1_UNIT'/; s/OFFHEAP_RESOURCE1_SIZE/'$OFFHEAP_RESOURCE1_SIZE'/' conf/tc-config*.xml \
```

Comment:
Does this work with the nested single quotes?

Reply:
yup, been working for a while !